src/client/src/rt-theme/ThemeState.tsx

```diff
@@ -42,50 +42,12 @@ class ThemeStateProvider extends React.Component<ThemeStateProps> {
     return <ContextConsumer children={this.renderThemeStateManager} />
   }
 
-  renderThemeStateManager = (context: ThemeStateValue) => {
```

Comment:
We lose the theme boundaries — but seeing how we weren't using them anyways, this works.